pybamm/expression_tree/parameter.py

```diff
@@ -48,19 +48,21 @@ class FunctionParameter(pybamm.Symbol):
 
     name : str
         name of the node
-    child : :class:`Symbol`
-        child node
+    inputs : dict
+        A dictionary with string keys and :class:`pybamm.Symbol` values representing
```

Comment:
maybe say what the string should be i.e. useful/descriptive names for the inputs